config/schema_config.yaml

```diff
@@ -754,17 +751,18 @@ upstream gene to variant association:
     holds between a sequence variant and a gene that is upstream to the variant
   represented_as: edge
   input_label: upstream_gene
+  output_label: snp_upstream_gene
```

Comment:
you better make `input_label: snp_upstream_gene` and `output_label: upstream_of`. The same applies for downstream